client/src/pages/login/applicantDash/applicant.jsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import './applicant.css';
+
+function Applicant() {
```

Comment:
Change to a class, because we will be working with states later on
```suggestion
class Welcome {
  render() {
    return <h1>Hello, {this.props.name}</h1>;
  }
}
```

Reply:
Ignore this